Product Card Landscape/product-card-landscape.css

```diff
@@ -0,0 +1,44 @@
+*{
+    margin: 0;
+    padding: 0;
+}
+html{
+    font-size:62.5%
+}
+
+body{
+    background: var(--half-white);
+    height: 100vh;
+}
+
+.product-landscape-container{
+    width: 50rem;
+    display: flex;
+    flex-direction: row;
```

Comment:
Suggestion: Since `flex-direction: row` is the default property, you can avoid giving that.